src/Microsoft.AspNet.Mvc.Core/ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -179,7 +179,12 @@ public override string Description
         {
             get
             {
-                return DisplayMetadata.Description;
+                if (DisplayMetadata.Description == null)
+                {
+                    return null;
+                }
+
+                return DisplayMetadata.Description();
```

Comment:
FYI only. This can be done in one line w/ C# 6.0: `return DisplayMetadata.Description?.Invoke();`